xtokens/src/lib.rs

```diff
@@ -248,92 +251,101 @@ pub mod module {
 			asset: MultiAsset,
 			dest: MultiLocation,
 			recipient: MultiLocation,
-			buy_order: Order<()>,
-		) -> Xcm<T::Call> {
-			WithdrawAsset {
-				assets: vec![asset],
+			dest_weight: u64,
```

Comment:
```suggestion
			dest_weight: Weight,
```
I would use Weight in every function